example/tests/test_errors.py

```diff
@@ -1,13 +1,33 @@
 import pytest
 from django.test import override_settings
 from django.urls import path, reverse
-from rest_framework import views
+from rest_framework import generics
 
 from rest_framework_json_api import serializers
 
 from example.models import Blog
 
 
+class AuthorModelMock:
+    __name__ = "author"
+
+    @property
+    def model(self):
+        return self
+
+    @property
+    def queryset(self):
+        return self
+
+    def create(self, *args, **kwargs):
+        pass
+
+    def get(self, pk):
+        if isinstance(pk, int):
+            return self
```

Comment:
Odd... I thought I'd commented here before but somewhat cannot find this comment anymore.

Did you consider using `example.models.Author` instead of this mock? Issue here is that in JSON:API the id is actually always a string so line 27 will never be covered (reason why CodeCov is failing).